src/concrete/ml/sklearn/base.py

```diff
@@ -1679,6 +1680,36 @@ class SklearnLinearRegressorMixin(SklearnLinearModelMixin, sklearn.base.Regresso
     """
 
 
+class SklearnSGDRegressorMixin(SklearnLinearRegressorMixin):
+    """A Mixin class for sklearn SGD regressors with FHE.
+
+    This class is used to create a SGD regressor class what can be exported
+    to ONNX using Hummingbird.
+    """
+
+    def _set_onnx_model(self, test_input: numpy.ndarray) -> None:
```

Comment:
would be great if you could create an issue that say that this should be removed once HB integrates the model (could you maybe add an issue on their side as well) + add a fixme here so that we don't forget about it !